### DISCUSSION TITLE
imagemagick Custom font is blurred

### DISCUSSION THREAD

**Discussion by a123likeyou** [ImageMagick/ImageMagick6#132] [Q&A]
The image font generated by using imagemagick to customize the font size and type is blurry, is there any solution?

The command is：Magick convert -size 200.0x50.0 -pointsize 16 -fill rgba(0,0,0,1.0) 
  -font C:\image-magick-fonts\5d4c258debc748cfac67a496d65f96e6.ttf  
  -background none -gravity center caption:"Kanbe Kotori/njkjkj " D:\output.png
![image](https://user-images.githubusercontent.com/22471187/106255815-76cfbc80-6255-11eb-959c-273a2af56986.png)

**Comment by urban-warrior** [comment]
Type `magick --version`.  Does the delegate list include `freetype`?  If not, download & install the Freetype delegate library and reconfigure, rebuild, and reinstall ImageMagick.  Do you get expected results now?
